ui/components/app/security-provider-banner-alert/security-provider-banner-alert.js

```diff
@@ -98,6 +121,9 @@ SecurityProviderBannerAlert.propTypes = {
 
   /** Name of the security provider */
   provider: PropTypes.oneOf(Object.values(SecurityProvider)),
+
+  /** Function to be called when the blockaid support link is clicked */
+  onClickBlockaidSupport: PropTypes.func,
```

Comment:
Let's rename this to `onClickSupport` or `onClickSupportLink`. This component was made to be generic to allow scaling and reuse. Coupling logic can lead to confusion and complications